Makefile

```diff
@@ -119,7 +119,7 @@ test: all
 
 lint:
 	# check for unused imports, the rest is done by black
-	flake8 --select=F401 src tools pyodide-build benchmark conftest.py docs
+	flake8 --select=F401 src tools pyodide-build benchmark conftest.py docs packages/matplotlib/src/
```

Comment:
@rth maybe we should be applying flake8 to micropip too?

Reply:
yes, but we can do that in a separate PR.

Reply:
Resolving since it is to be done in a separate PR.